src/POMtests/AddToCart.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, test } from "@playwright/test";
+import LoginPage from "../pages/loginPage";
+import HomePage from "../pages/homePage";
+import SpecialHotPage from "../pages/specialHotPage";
+
+const email = "MilaKamila@mailinator.com";
+const password = "MilaKamila@123";

```

Comment:
why do we need these 2 variables (email, password) here?

Reply:
made changes